checkov/common/runners/object_runner.py

```diff
@@ -138,3 +161,32 @@ def _change_files_path_to_relative(report: Report) -> None:
         for record in report.get_all_records():
             record.file_path = record.file_path.replace(os.getcwd(), "")
             record.resource = record.resource.replace(os.getcwd(), "")
+
+    def _get_triggers(self, result: Tuple[Dict[str, Any], Dict[str, Any]]) -> set[str]:
+        triggers_set = set()
+        triggers = result[0].get(True)
+        try:
+            if triggers:
+                for key in triggers.keys():
+                    if key != START_LINE and key != END_LINE:
+                        triggers_set.add(key)
```

Comment:
you can write this as a set comprehension `{... for ... if ...}` this has better performance